api/projects/models.py

```diff
@@ -27,6 +27,10 @@ class Project(models.Model):
         default=False,
         help_text="If true will exclude flags from SDK which are " "disabled",
     )
+    enable_dynamo_db = models.BooleanField(
+        default=False,
+        help_text="If true will fetch identity data(shown in the dashboard) from dynamodb",
```

Comment:
This isn't true, right? At the moment, it's just going to store environments in dynamodb. Nothing to do with identities.